debian/snapd.postrm

```diff
@@ -3,7 +3,19 @@
 set -e
 
 if [ "$1" = "purge" ]; then
-   # FIXME: should we try to remove all snaps, mount points, services,
-   #        mount units that got created as well?
-   rm -f /var/lib/snapd/state.json
-fi
\ No newline at end of file
+
+    for snap in /var/lib/snapd/snaps/*.snap; do
+        base="$(basename $snap)"
+        # mount units
+        mangledbase="$(echo $base|tr _ -)"
+        mountunit="$(systemd-escape -p "${mangedbase%%.snap}".mount)"
```

Comment:
Typo in var name.

Related but off-topic note: can we change these filenames to not escape?  Our current conventions for snap and app name make them safe.